utils/validation/program.ts

```diff
@@ -0,0 +1,29 @@
+import { createProgramInput } from "models/Program";
+
+/**
+ * validateProgram takes programData of type Program and validates the data
+ * @param newProgramData - data corresponding to new program
+ * @returns true: validated, false: not validated
+ */
+export function validateCreateProgram(
```

Comment:
I think I said this in another PR but it would be good to return a specific error regarding what was invalid about the input. This way, it'll provide a much more descriptive error message to the frontend.